static/app/components/core/layout/stack.mdx

```diff
@@ -0,0 +1,243 @@
+  Basic stack content
+</Stack>
+```
+
+### Stack Properties
+
+Stack provides a focused set of layout properties: `direction` (defaults to 'column'), `gap`, `justify`, and `align`. These properties influence the layout of its children while maintaining simplicity.
+
+Like other layout components, `Stack` inherits all spacing props like `m`, `p`, `mt`, `mb`, `ml`, `mr`, `pt`, `pb`, `pl`, `pr` and implements responsive props so that the layout can be changed per breakpoint.
+
+#### Column Direction (Default)
+
+<Storybook.Demo>
+  <Stack gap="md" justify="center" align="center" padding="md">
+    <Container padding="md" border="primary" radius="md" background="primary">
+      Item 1
+    </Container>
+    <Container padding="md" border="primary" radius="md" background="primary">
+      Item 2
+    </Container>
+    <Container padding="md" border="primary" radius="md" background="primary">
+      Item 3
+    </Container>
+  </Stack>
+</Storybook.Demo>
+```jsx
+<Stack gap="md" justify="center" align="center" padding="md">
+  <div>Item 1</div>
+  <div>Item 2</div>
+  <div>Item 3</div>
+</Stack>
+```
+
+#### Row Direction
+
+<Storybook.Demo>
+  <Stack direction="row" gap="md" justify="between" align="center" padding="md">
+    <Container padding="md" border="primary" radius="md" background="primary">
+      Item 1
+    </Container>
+    <Container padding="md" border="primary" radius="md" background="primary">
+      Item 2
+    </Container>
+    <Container padding="md" border="primary" radius="md" background="primary">
+      Item 3
+    </Container>
+  </Stack>
+</Storybook.Demo>
+```jsx
+<Stack direction="row" gap="md" justify="between" align="center" padding="md">
+  <div>Item 1</div>
+  <div>Item 2</div>
+  <div>Item 3</div>
+</Stack>
+```
+
+#### Spacing
+
+The `Stack` `gap` property follows the same spacing system as other layout components.
+
+<Storybook.Demo>
+  {['xs', 'sm', 'md', 'lg', 'xl', '2xl'].map(size => (
+    <Stack direction="column" gap="sm" key={size}>
+      <strong>{size} gap</strong>
+      <Stack m="md" gap={size} align="center">
+        <Container padding="md" border="primary" radius="md" background="primary">
+          Item 1
+        </Container>
+        <Container padding="md" border="primary" radius="md" background="primary">
+          Item 2
+        </Container>
+        <Container padding="md" border="primary" radius="md" background="primary">
+          Item 3
+        </Container>
+      </Stack>
+    </Stack>
+  ))}
+</Storybook.Demo>
+```jsx
+<Stack m="md" gap="xs">
+  <div>Item 1</div>
+  <div>Item 2</div>
+  <div>Item 3</div>
+</Stack>
+```
+
+#### Responsive Props
+
+All props support responsive values using breakpoint objects. Breakpoints are: `xs`, `sm`, `md`, `lg`, `xl`, `2xl`.
+
+Example of a responsive stack that uses a static gap, but changes direction based on the breakpoint.
+
+<Storybook.Demo>
+  <Stack direction={{xs: 'column', sm: 'row', md: 'column'}} gap="md" p="md">
```

Comment:
just questioning the value of this example here, like why would only `sm` be `row` and everything above it would be column again ?

Reply:
Let me just change this to column and row, that's probably enough